processor/processor.go

```diff
@@ -3283,11 +3282,15 @@ func (proc *Handle) handlePendingGatewayJobs(partition string) bool {
 
 	var transMessage *transformationMessage
 	var err error
-	transMessage, err = proc.processJobsForDest(partition, subJob{
-		subJobs:       unprocessedList.Jobs,
-		hasMore:       false,
-		rsourcesStats: rsourcesStats,
-	})
+	transMessage, err = proc.generateTransformationMessage(proc.processJobsForDest(
```

Comment:
not related to this PR. but there's a possibility of duplicate persistence of jobs in events schema and archival?

Reply:
It's possible yeah